### PR TITLE
chore: clean up Marketo implementation for file upload et al

### PR DESCRIPTION
**Pull request by utsabc**
## What are the changes introduced in this PR?

This PR aims to clean up the fileUpload, poll, and jobstatus functionalities of Marketo Bulk Upload Implementation, as this implementation have been migrated to rudder server via following [PR](https://github.com/rudderlabs/rudder-server/pull/5114)

## What is the related Linear task?

Resolves INT-765

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### Any technical or performance related pointers to consider with the change?

N/A

@coderabbitai review

<hr>

### Developer checklist

- [ ] My code follows the style guidelines of this project

- [ ] **No breaking changes are being introduced.**

- [ ] All related docs linked with the PR?

- [ ] All changes manually tested?

- [ ] Any documentation changes needed with this change?

- [ ] Is the PR limited to 10 file changes?

- [ ] Is the PR limited to one linear task?

- [ ] Are relevant unit and component test-cases added in **new readability format**?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Comment by devops-github-rudderstack** [comment]
Test report for this run is available at: https://test-integrations-dev.s3.amazonaws.com/integrations-test-reports/rudder-transformer/3872/test-report.html

**Comment by koladilip** [comment]
so marketo new implementation is stable enough to remove this?

**Comment by utsabc** [comment]
> so marketo new implementation is stable enough to remove this?

Yes nothing unusual is observed

**Review by koladilip — APPROVED** [review]

File: src/v0/destinations/marketo_bulk_upload/util.js [hunk 1]
```diff
@@ -1,436 +0,0 @@
...
-        actualDataType !== expectedDataType
-      ) {
-        mismatchedFields[job_id] = `invalid ${paramName}`;
-      }
-    });
-  });
-  return mismatchedFields;
-};
-
-module.exports = {
```
Comment (krishna2020) [inline src/v0/destinations/marketo_bulk_upload/util.js:426]: extend eslint config to find missing imports - if any of these utils are being referenced anywhere else

Comment (utsabc) [reply src/v0/destinations/marketo_bulk_upload/util.js:426]: Manually verfied no referecens are present, the eslint changes were not actually flagging need properly check later.

File: src/util/fetchDestinationHandlers.ts
Comment (krishna2020) [inline src/util/fetchDestinationHandlers.ts:1]: Can't this file be deleted?

**Review by krishna2020 — APPROVED** [review]

File: src/util/fetchDestinationHandlers.ts
Comment (utsabc) [reply src/util/fetchDestinationHandlers.ts:1]: No its being referenced in core service layer, I want to raise a separate PR for that clean up

**Pull request merged**